### PR TITLE
Fix compiler errors with gcc-7

### PR DESCRIPTION
**Pull request by HolyShitMan**
Compiling with gcc-7, two errors occurred:
Firstly, an include was missing (commit 5127067) and secondly the compiler was complaining about dangling else ambiguity in tests (commit 0d407f0) when EXPECT_xx was used in if-conditions without surrounding brackets.

### CODE REVIEW

File: tests/json_ut.cpp [hunk 1]
```diff
@@ -48,8 +48,9 @@ TEST(JSON, test)
 
         for (auto it = item.begin(); it != item.end(); it++)
         {
-            if (it.key() == "OP")
+            if (it.key() == "OP") {
```
Comment (qiluo-msft) [inline tests/json_ut.cpp]: >{ [](start = 34, length = 1)

move to next line.

File: tests/json_ut.cpp [hunk 2]
```diff
@@ -59,10 +60,12 @@ TEST(JSON, test)
                 {
                     for (auto subit = subitem.begin(); subit != subitem.end(); subit++)
                     {
-                        if (subit.key() == "test_field_1")
+                        if (subit.key() == "test_field_1") {
```
Comment (qiluo-msft) [inline tests/json_ut.cpp]: >{ [](start = 59, length = 1)

the same.

File: tests/json_ut.cpp [hunk 3]
```diff
@@ -59,10 +60,12 @@ TEST(JSON, test)
...
                             EXPECT_EQ(subit.value(), "test_value_1");
-                        if (subit.key() == "test_field_2")
+                        }
+                        if (subit.key() == "test_field_2") {
```
Comment (qiluo-msft) [inline tests/json_ut.cpp]: >{ [](start = 59, length = 1)

the same.

File: tests/redis_piped_ut.cpp [hunk 4]
```diff
@@ -364,8 +364,12 @@ TEST(Table, piped_test)
         {
             string value_1 = "1", value_2 = "2";
             cout << " " << fvField(fv) << ":" << fvValue(fv) << flush;
-            if (fvField(fv) == "field_1") EXPECT_EQ(fvValue(fv), value_1);
-            if (fvField(fv) == "field_2") EXPECT_EQ(fvValue(fv), value_2);
+            if (fvField(fv) == "field_1") {
```
Comment (qiluo-msft) [inline tests/redis_piped_ut.cpp]: >{ [](start = 42, length = 1)

the same.

File: tests/redis_ut.cpp [hunk 5]
```diff
@@ -196,8 +196,12 @@ void TableBasicTest(string tableName)
         {
             string value_1 = "1", value_2 = "2";
             cout << " " << fvField(fv) << ":" << fvValue(fv) << flush;
-            if (fvField(fv) == "field_1") EXPECT_EQ(fvValue(fv), value_1);
-            if (fvField(fv) == "field_2") EXPECT_EQ(fvValue(fv), value_2);
+            if (fvField(fv) == "field_1") {
```
Comment (qiluo-msft) [inline tests/redis_ut.cpp]: >{ [](start = 42, length = 1)

the same.

**Comment by HolyShitMan** [comment]
Fixed coding style and squashed commit into corresponding one.

**Pull request merged**